src/edu/cmu/cs/sasylf/util/ErrorHandler.java

```diff
@@ -1,11 +1,9 @@
 package edu.cmu.cs.sasylf.util;
 
+import edu.cmu.cs.sasylf.term.FreeVar;

```

Comment:
Are there any real changes to this file?  They seem to be only reformatting changes?